Expensify/App
# Fix the text style in about page

@shawnborton & @bondydaa  

### Details
The text in the about page wasn't using our regular font size. 

### Fixed Issues
$ N/A

### QA Steps

1. Click on your user profile image to open the Settings menu
2. Go to "About"
3. Verify that the text is in a normal font size

### Tested On


### Screenshots


#### Web

![Screen Shot 2021-08-10 at 10 47 42](https://user-images.githubusercontent.com/1805746/128840810-8c770c25-6064-4e00-baca-04b657bc3e84.png)


#### Mobile Web

![Screen Shot 2021-08-10 at 10 57 49](https://user-images.githubusercontent.com/1805746/128840834-8565a416-1f1d-4622-aedf-6418e61fe118.png)


#### Desktop


#### iOS


#### Android
